### DISCUSSION TITLE
Connect to other Databases

### DISCUSSION THREAD

**Discussion by omaromp2** [NativePHP/laravel#316] [Q&A] [answered]
Is it possible to configure a NativePhp app with a local DB other than sqlite? I mean, what if I want to build an ETL connector that uses eloquent.

**Comment by simonhamp** [comment, marked as answer]
@omaromp2 there are no plans to add other DB drivers to the out-of-the-box PHP binaries at this stage.

You will need to build [your own static PHP binary](https://static-php.dev/) for the platforms you wish to support.

**Reply by omaromp2** [reply]
Thanks @simonhamp! 
I'll look into it! 
Thank you for the quick reply!
